README.md

```diff
@@ -304,6 +304,31 @@ export default {
 
 Components having the same name in `~/components` will overwrite the one in `my-theme/components`, learn more in [Overwriting Components](#overwriting-components). The lowest value will overwrite.
 
+#### prefetch/preload
+
+- Type: `Boolean/Number`
+- Default: `false`
+
+These properties are used for Wepack's magic comments, learn more in [Wepack's official documentation](https://webpack.js.org/api/module-methods/#magic-comments).
```

Comment:
It would be nice to mention it is effective only for `Lazy*` components. During development, we don't enable loader so it affects every component but in production, detected components are directly imported via `installComponents`

Reply:
Thank you, I updated README by 6c33178aa9d03d82d482fa39bf03c86c905c17b8.